internal/featuresupport/support_levels_list.go

```diff
@@ -105,4 +105,38 @@ var SupportLevelsList = models.FeatureSupportLevels{
 			},
 		},
 	},
+	&models.FeatureSupportLevel{
+		OpenshiftVersion: "4.11",
+		// TODO: This list has been copied from 4.10 as-is just so I can add
+		// usage.SingleNodeExpansion as a supported feature for 4.11. This
+		// means I don't know know for sure whether all the features have the
+		// same support level in 4.11 as they had in 4.10, so take this list
+		// with a grain of salt and don't be afraid to change it if you know
+		// it should be different.
+		Features: []*models.FeatureSupportLevelFeaturesItems0{
+			// Supported
+			{
+				FeatureID:    usageNameToID(usage.SingleNodeExpansion),
```

Comment:
I think you should add it as unsupported to all other versions, I'm not sure what is the default behavior in the UI
@jkilzi what is the default?

Reply:
Sounds incredibly unnecessary & verbose. If a feature is not mentioned for a particular version, it should be assumed to be unsupported. Even if the UI is not prepared for this behavior, our service should do it automatically

Reply:
I'll resolve it in a separate PR